docs/settings.rst

```diff
@@ -434,8 +434,10 @@ The number of days an activation token is valid, env ``ACTIVATE_TTL``.
 ^^^^^^^^^^^^^^
 
 .. versionadded:: 1.5
+.. versionchanged:: 2.4
+    Now required as ``flask.session`` will be used for some authentication
 
-Default: ``''``
+Default: Hardcoded value, which should not be used in production!
```

Comment:
I think we shouldn't have any default here, this must be set via environment.

Reply:
If we don't put a default, tests/fireq/customer instances will all start failing.

I can change this if required, but will need to update all the affected tests/instances as well (probably under a separate ticket as it will be a big task)

Reply:
I can handle fireq, that's not a big deal, for tests we can set something hardcoded, but otherwise we should require this to be set

Reply:
but we can probably keep it, seems consistent with newsroom

Reply:
I actually copied this code from NewsHub (except generated a new secret key), as I wanted to see how we handle it there.

I do agree it's not the best to have a hard-coded default for this, and not defining this value is an invalid config which should not proceed to run.

What if we merge this for now and create a ticket to tackle both Superdesk & NewsHub together?

Reply:
yep makes sense